lib/models/blueprint.js

```diff
@@ -894,6 +894,60 @@ Blueprint.prototype.addPackagesToProject = function(packages) {
 };
 
 /**
+  Used to remove a package from the projects `package.json`.
+
+  Generally, this would be done from the `afterInstall` hook, to
+  ensure that any package conflicts can be resolved before the
+  addon is used.
+
+  @method removePackageFromProject
+  @param {String} packageName
+  @return {Promise}
+*/
+Blueprint.prototype.removePackageFromProject = function(packageName) {
+  var packageObject = {name: packageName};
+
+  return this.removePackagesFromProject([packageObject]);
+};
+
+/**
+  Used to remove multiple packages from the projects `package.json`.
+
+  Generally, this would be done from the `afterInstall` hook, to
```

Comment:
@abuiles yup updated sorry for the back and forwards with this, it's not acceptable I am sorry for that.